content/ko/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -148,6 +148,24 @@ kube-scheduler는 각 파드에 대해 구성된 리소스 [요청과 제한](/k
 메트릭은 HTTP 엔드포인트 `/metrics/resources`에 노출되며 스케줄러의 `/metrics` 엔드포인트
 와 동일한 인증이 필요하다. 이러한 알파 수준의 메트릭을 노출시키려면 `--show-hidden-metrics-for-version=1.20` 플래그를 사용해야 한다.
 
+## 메트릭 비활성화
+
+커맨드 라인 플래그 `--disabled-metrics` 를 통해 지표를 명시적으로 끌 수 있다. 예를 들어, 메트릭이 성능 문제를 일으키는 경우 이 방법이 필요할 수 있다. 입력은 비활성화된 메트릭 항목의 목록이다(예: `--disabled-metrics=metric1,metric2`).
```

Comment:
```suggestion
커맨드 라인 플래그 `--disabled-metrics` 를 통해 메트릭을 명시적으로 끌 수 있다. 이 방법이 필요한 이유는 메트릭이 성능 문제를 일으키는 경우을 예로 들 수 있다. 입력값은 비활성화되는 메트릭 목록이다(예: `--disabled-metrics=metric1,metric2`).
```
메트릭을 계속 사용하고 있어서, 해당 항목만 지표로 바꾸기 보다는 일관성을 유지하면 좋을 것 같습니다.. ^^
그리고 조금 자연스럽게 변경해보았습니다.

```
This may be desired if, for example, a metric is causing a performance problem. The input is a list of disabled metrics (i.e. `--disabled-metrics=metric1,metric2`).
```

Reply:
반영하겠습니다~